src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -245,7 +252,8 @@ HttpStatusCode mongoQueryContext
   QueryContextResponse*                responseP,
   const std::string&                   tenant,
   const std::vector<std::string>&      servicePathV,
-  std::map<std::string, std::string>&  uriParams
+  std::map<std::string, std::string>&  uriParams,
+  long long*                           countP
```

Comment:
Just to clarify, **countP** is both input and output.
If not NULL, then the count is asked for (mongo operation) and returned in *countP.